biome.jsonc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
i think this config may actually be a replacement for eslint-plugin-prettier; not exactly sure they are identical tho

Reply:
🔗 https://github.com/SrBrahma/eslint-config-biome?tab=readme-ov-file#%E2%84%B9%EF%B8%8F-info